src/scripts/dungeons/Dungeon.ts

```diff
@@ -3606,6 +3740,14 @@ dungeonList['Sealed Chamber'] = new Dungeon('Sealed Chamber',
         new DungeonBossPokemon('Regirock', 4500000, 20, {requirement: new QuestLineStepCompletedRequirement('The Three Golems', 8)}),
         new DungeonBossPokemon('Regice', 4500000, 20, {requirement: new QuestLineStepCompletedRequirement('The Three Golems', 8)}),
         new DungeonBossPokemon('Registeel', 4500000, 20, {requirement: new QuestLineStepCompletedRequirement('The Three Golems', 8)}),
+        new DungeonBossPokemon('Togepi (Flowering Crown)', 2700000, 23, {
```

Comment:
I think here should be 'Torchic (Egg)'

Reply:
And it seems that `new PokemonDefeatedSelectNRequirement('Pikachu (Easter)', 4, 6, 1)` is missing, causing Pikachu to disappear in Kanto